.gitignore

```diff
@@ -6,6 +6,8 @@ build/*
 
 npm-debug.log*
 *.sublime*
+*~
+tags
```

Comment:
Could you elaborate why we should add these two?

Reply:
some text editors suffix backup files with `~` and use the file `tags` to store information so you can jump to function definitions. But taggers don't work that well with Javascript it seems, so we could also get rid of the `tags`.